articles/architecture-scenarios/implementation/b2c/authentication.md

```diff
@@ -0,0 +1,131 @@
+---
+title: User Authentication
+description: Understand how authentication works in your CIAM implementation.
+toc: true
+topics:
+    - b2c
+    - ciam
+    - authentication
+    - universal-login
+contentType: concept
+useCase:
+    - authentication
+---
+# User Authentication
+
+In order to provide services to your users, you must be able to identify who those users are. This process is called authentication. You must provide your users with a way to verify that they are who they are, and there are a number of ways to do this; via social media accounts, username/password, and passwordless authentication to name but a few. In many cases it is also recommended that you go beyond a first factor for authenticating the user add a second factor as well.
+
+## Design considerations
+
+When designing the authentication experience, you will determine:
+
+* Where users will enter their credentials
+* How you will keep user credentials safe
+* How you will maintain your authentication system
+* How you will provide password authentication for your users
+* If you will integrate social logins for my users
+* How you will prevent hackers from trying to log in as your users
+* How you want to implement authentication in different kinds of applications using Auth0
+* If you want to make login easy for your users when they come from different language backgrounds
+* How you will provide a good user experience as you migrate away from my legacy authentication system
+
+The answers to these questions, and others, are detailed below in sections on [Universal Login](#universal-login), [user authentication](#user-authentication), [anomaly detection](#anomaly-detection), and [application integration](#application-integration).
```

Comment:
This leaves it to the reader to have to figure out where the answers actually are. In line with our more prescriptive strategy we opted to explicitly guide the reader to where they need to go; hence the narrative from the original document (https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.jw0r0xkd9gjo). Can we please restore the original narrative for this section. Thank you 👍

Reply:
Yes, I thought I did that. I'll go back and look at it again.